diff --git i/third_party/manifest-parser.js w/third_party/manifest-parser.js
index 5ac9d72..b9e7ead 100644
--- i/third_party/manifest-parser.js
+++ w/third_party/manifest-parser.js
@@ -17,7 +17,10 @@
 'use strict';
 
 const url = require('url');
-const validateColor = require('./web-inspector').Color.parse;
+
+global.WebInspector = {}; // the global is unfortunate, but necessary
+require('./Color.js');
+const validateColor = global.WebInspector.Color.parse;
 
 const ALLOWED_DISPLAY_VALUES = [
   'fullscreen',
